src/jit.h

```diff
@@ -51,6 +51,24 @@ void jit_translate(riscv_t *rv, block_t *block);
 typedef void (*exec_block_func_t)(riscv_t *rv, uintptr_t);
 
 #if RV32_HAS(T2C)
-void t2c_compile(block_t *block, uint64_t mem_base);
+void t2c_compile(riscv_t *, block_t *);
 typedef void (*exec_t2c_func_t)(riscv_t *);
+
+#if RV32_HAS(JIT_CACHE)
+/* The jit-cache records the program counters and the entries of executable
```

Comment:
"overridden" is not precise in terms of CPU cache. In general, we would say, cache replacement or cache eviction. It is not a matter to override something. Instead, cache is selective to all available candidates.